e2e/tests/ui_tests/evidence_upload/1v1UnspecEU_test.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-unused-vars */
+
+const config = require('../../../config.js');
+const mpScenario = 'ONE_V_ONE';
+const judgeUser = config.judgeUserWithRegionId1;
+let caseId;
+
+Feature('1v1 Unspec evidenceUpload');
+
+Scenario('EvidenceUpload @e2e-eu', async ({I, api}) => {
+  await api.createClaimWithRepresentedRespondent(config.applicantSolicitorUser, mpScenario, '11000');
+  await api.amendClaimDocuments(config.applicantSolicitorUser);
+  await api.notifyClaim(config.applicantSolicitorUser);
+  await api.notifyClaimDetails(config.applicantSolicitorUser);
+  await api.defendantResponse(config.defendantSolicitorUser, mpScenario);
+  await api.claimantResponse(config.applicantSolicitorUser, mpScenario, 'AWAITING_APPLICANT_INTENTION', 'FOR_SDO');
+  await api.createSDO(judgeUser, 'CREATE_FAST_NO_SUM');
+
+  caseId = await api.getCaseId();
+  await I.login(config.applicantSolicitorUser);
```

Comment:
Move this test to be covered in https://github.com/hmcts/civil-ccd-definition/blob/master/e2e/tests/ui_tests/damages/1v2CreateClaim_DiffSol_test.js#L131 so that we dont need to run same steps every time in e2e tests

Reply:
and here as well https://github.com/hmcts/civil-ccd-definition/blob/master/e2e/tests/ui_tests/lrspec/1v2CreateClaim-LRspec_DiffSol_test.js#L73
as another scenario for lr spec journey